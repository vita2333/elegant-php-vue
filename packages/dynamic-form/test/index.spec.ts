// import { mount, Wrapper } from '@vue/test-utils'
// import component, { DynamicFormProps } from '../index'
// import './demo.spec'
//
// describe('dynamic-form', () => {
//   let wrapper: Wrapper<Vue>
//   const propsData: Required<DynamicFormProps> = {
//     form: 'new form'
//   }
//   beforeEach(() => {
//     wrapper = mount(component, {
//       propsData
//     })
//   })
//   it('should render correct', () => {
//     expect(wrapper).toMatchSnapshot()
//   })
//   it('renders props.msg when passed', () => {
//     expect(wrapper.text()).toMatch(propsData.form)
//   })
// })
