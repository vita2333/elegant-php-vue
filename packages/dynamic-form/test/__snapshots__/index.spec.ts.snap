// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dynamic-form should render correct 1`] = `
<div class="dynamic-form">
  <h1>component-jsx</h1>
  <p>new form</p>
</div>
`;
